cloud/client.go

```diff
@@ -56,6 +56,7 @@ type Client struct {
 	requestID                string
 	installationName         string
 	logstreamAddressOverride string
+	serverConnTimeout        int
```

Comment:
can we make this a duration here instead?
```suggestion
	serverConnTimeout        time.Duration
```